git-hg/add-branch.sh

```diff
@@ -2,63 +2,63 @@
 
 echo "Common defs"
 
+# shellcheck disable=SC1091
 . import-common.sh
 
 echo "Enter hg"
 
 cd hg || exit 1
 
-bpath=$1
+echo "bpaths=(ls -d -1 */*)"
 
```

Comment:
I'm not sure that `ls -d -1 */*` is what you want to do here?  That lists all files and dirs (`-d` doesn't quite do what you might expect?)